content/reference/sql/access-management.md

```diff
@@ -0,0 +1,289 @@
+
+This leads to an interaction with the [privilege file](#privilege-file) that should be noted.
+When the privilege file is specified, it will be created as an empty file if it does not exist.
+As soon as any statement that modifies the grant tables (`CREATE USER`, `GRANT`, `REVOKE`, etc.) executes, the users and all privileges will save to the privilege file.
+This **includes** the super account as defined by the user and password arguments, therefore it is recommended that the super account is deleted after all users are set up, or it is given a strong password.
+On subsequent server starts, if the [privilege file](#privilege-file) _contains any data_, the user and password arguments are fully ignored.
+This behavior was chosen so that server should always have at least one user that a client may log into, otherwise the server would be completely inaccessible.
+
+## Editing Users
+
+Editing users and their privileges must be done from [`sql-server`](../cli.md#dolt-sql-server) mode.
+You may also directly edit the [privilege file](#privilege-file), which is detailed below.
+Dolt comes with a MySQL client built-in, which is the [`sql-client`](../cli.md#dolt-sql-client) command.
+In addition to allowing access to a running server, the client command also has the `--dual` flag which runs a server in the background, while the foreground automatically connects to the background server using the given port, etc.
+
+Importantly, as described in the [previous section](#user-and-password-arguments), if a non-empty privilege file is provided, then the `--user` and `--password` arguments (also available via a [YAML configuration file](configuration.md)) _only_ function as login credentials.
+Otherwise, the arguments handle both the creation of a super account **and** login credentials.
+
+## Privilege File
+
+The privilege file contains a JSON object, designed to be editable by anyone inclined to do so.
+The structure is straightforward, although it is guaranteed to change in the future as more features are added (thus increasing the data that needs to be persisted).
+Below is an example file representing the following properties (formatted for easier viewing):
+- The default super account is `root`
+  - May only be connected to from `localhost`
+- There is a role named `test_role`
+  - Has the privileges `SELECT`, `INSERT`, `UPDATE`, and `DELETE` on all databases and tables
+  - Roles are the same as users except for a special boolean being set, and are locked to prevent being used for logging in
+- There is a user named `test_user`
+  - Can be connected to from any IP
+  - Does not contain a password
+  - Has the `DROP` privilege on the table `some_tbl` in the database `some_db`
+  - Has both the `CREATE` and `DROP` privileges on the `other_db` database
+- The role `test_role` has been granted to `test_user`
+- The `"Columns"` field is present, but is currently unused (example will be updated when column-level privileges are supported)
```

Comment:
Nice to call out that although Columns is present, it's unused. I don't think you need to call out the "(example will be updated...)" part though.